web-app/package.json

```
@@ -49,7 +50,8 @@
     "lint": "eslint 'src/app/**/*.{js,ts,tsx}'",
     "lint:fix": "eslint 'src/app/**/*.{js,ts,tsx}' --fix",
     "cypress:run": "cypress run",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "generate:api-types": "npx openapi-typescript ../docs/DatabaseCatalogAPI.yaml -o src/app/services/feeds/types.ts"
```

[picky]: I noticed that `openapi-typescript` generates strings with double quotes instead of single quote. To avoid running `yarn lint:fix` every time this command is executed, let's add the lint fix to the command itself:
```suggestion
    "generate:api-types": "npx openapi-typescript ../docs/DatabaseCatalogAPI.yaml -o src/app/services/feeds/types.ts  && eslint src/app/services/feeds/types.ts --fix"
```

---

great idea. done